src/components/ContentUploader/UploaderUtils.js

```diff
@@ -0,0 +1,7 @@
+class UploaderUtils {
+    static isMultiputSupported() {
```

Comment:
can this be a function in `src/util/uploads.js`?

Reply:
Didn't notice that file existed, oops! will move it there :)

Reply:
or maybe `webcrypto.js`